packages/soupault/soupault.1.8.0/opam

```diff
@@ -42,3 +42,11 @@ depends: [
   "tsort"
   "lua-ml" {>= "0.9.1"}
```

Comment:
This packages version does not exist. This is why no errors showed on the original PR

Reply:
It didn't exist at the time of writing, which is why I left a note about it in #15682. It does exist now that #15668 is merged: https://opam.ocaml.org/packages/lua-ml/lua-ml.0.9.1/
Or I'm missing something?